WalletWasabi/BranchNBound/BranchAndBound.cs

```diff
@@ -0,0 +1,108 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.BranchNBound
+{
+	public class BranchAndBound : Selector
+	{
+		private Money _costOfHeader = Money.Satoshis(0);
+		private Money _costPerOutput = Money.Satoshis(0);
+		private int _bnbTryLimit = 5;
+		private Random _random = new();
```

Comment:
Why is this 5?

Reply:
[This](https://github.com/zkSNACKs/WalletWasabi/pull/6825#discussion_r767616910) comment.
That 5 was just for testing purposes, it should be replaced with a bigger number, but im not sure about that number.